### DISCUSSION TITLE
do all weather stations have precipitation data?

### DISCUSSION THREAD

**Discussion by hstoebel** [weather-gov/api#626] [Q&A] [answered]
I'm trying to get precipitation data from a weather station on a day where I am 99% sure it rained but I'm being told there was no rain on that day The url I'm calling is: https://api.weather.gov/stations/KRGA/observations?start=2023-03-30T00%3A00%3A00Z&end=2023-03-31T12%3A00%3A00Z.

Is it possible this weather station just doesn't track rainfall? Am I doing something wrong?

**Comment by akrherz** [comment, marked as answer]
A couple places to check for metadata to see if a site has a precipitation sensor: 

https://www.ncei.noaa.gov/access/homr/
https://www.faa.gov/air_traffic/weather/asos  indicates RGA is a AWOS-3, so it isn't maintained to the quality of a NWS/FAA ASOS, so it could be plugged or not have the sensor or about anything else under the sun.

My database has no precipitation reported from this site since its 2019 start date.
